lib/oban/notifiers/pg.ex

```diff
@@ -50,7 +50,7 @@ defmodule Oban.Notifiers.PG do
   @impl Notifier
   def notify(server, channel, payload) do
     with %{conf: conf} <- get_state(server) do
-      pids = :pg.get_members(__MODULE__, conf.prefix)
+      pids = :pg.get_members(__MODULE__, {conf.name, conf.prefix})
```

Comment:
This would be a breaking change for any app that relies on connecting different apps that share an `oban_jobs` table. Instead, I think the solution is to pass a `namespace` option through and have it fall back to `conf.prefix`.

In application config it would look like this:

```elixir
notifier: {Oban.Notifiers.PG, namespace: {Oban, "public"}}
```

WDYT?

Reply:
Funny enough @mbaleczny suggested something like that as well.

Reply:
@sorentwo I don't see how this would effect two apps connected to the same database using the same table. If both have the same `name:` then they will still communicate as normal but if the `name:` is different, then they won't communicate.

Either way, I do agree that this should probably be a configuration done with the `:notifier`. I don't know where all of that would need to change as my understanding is that `:notifier` is expected to be a module only.

Reply:
> I don't see how this would effect two apps connected to the same database using the same table. If both have the same name: then they will still communicate as normal but if the name: is different, then they won't communicate.

That's the issue—if people are relying on the fact that two instances with different names (`Oban.A` and `Oban.B`) can communicate, then this would break their app.

> I don't know where all of that would need to change as my understanding is that :notifier is expected to be a module only.

I can make the changes. It will look roughly like https://github.com/sorentwo/oban_notifiers_phoenix 🙂

Reply:
Cool, I'll close this PR then.